### PR TITLE
[#2 Kill Task functionality] Task kill actions

### PR DESCRIPTION
**Pull request by kennyt**
px

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

**Comment by rcorral** [comment]
Rebase please!

**Comment by kennyt** [comment]
rebased

File: src/js/components/KillTaskModal.js [hunk 1]
```diff
@@ -17,6 +19,7 @@ class KillTaskModal extends mixin(StoreMixin) {
     super(...arguments);
 
     this.store_listeners = [
+      {name: 'marathon', events: ['taskKillSuccess'], suppressUpdate: true}
```
Comment (rcorral) [inline src/js/components/KillTaskModal.js]: @kennyt can you add `taskKillError` to the ServerErrorModal?

Comment (kennyt) [inline src/js/components/KillTaskModal.js]: done https://github.com/dcos/dcos-ui/pull/576/commits/65d4e67556be96d171b0fe37e09e5c1ae873488b

**Pull request merged**